samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -254,19 +254,29 @@ def _load_pipeline_bootstrap_resources() -> Tuple[List[str], Dict[str, str]]:
     return stage_names, context
 
 
-def _copy_dir_contents_to_cwd_fail_on_exist(source_dir: str) -> List[str]:
-    copied_file_paths: List[str] = []
+def _copy_dir_contents_to_cwd(source_dir: str) -> List[str]:
+    file_paths: List[str] = []
+    existing_file_paths: List[str] = []
     for root, _, files in os.walk(source_dir):
         for filename in files:
             file_path = Path(root, filename)
             target_file_path = Path(".").joinpath(file_path.relative_to(source_dir))
             LOG.debug("Verify %s does not exist", target_file_path)
             if target_file_path.exists():
-                raise PipelineFileAlreadyExistsError(target_file_path)
-            copied_file_paths.append(str(target_file_path))
+                existing_file_paths.append(str(target_file_path))
+            file_paths.append(str(target_file_path))
+    if existing_file_paths:
+        click.echo("\nThe following files already exist:")
+        for existing_file_path in existing_file_paths:
+            click.echo(f"\t- {existing_file_path}")
+        if not click.confirm("Do you want to override them?"):
+            target_dir = str(Path(PIPELINE_CONFIG_DIR, "generated-files"))
+            osutils.copytree(source_dir, target_dir)
+            click.echo(f"All files are saved to {target_dir}.")
+            sys.exit(0)
```

Comment:
Why are we exiting here instead of returning? I am assuming we are returning list of path where source dir is copied, right? We can return pipeline config dir path.